nginx/start

```diff
@@ -25,11 +25,10 @@ else
 	nginx_template=/etc/nginx/conf.d/scalelite.template
 fi
 
-# apply SCALELITE_API port from environment variable or use default port 3000
-$SCALELITE_API_PORT=${SCALELITE_API_PORT:-3000}
-envsubst '$SCALELITE_API_PORT' <$nginx_template >/etc/nginx/conf.d/scalelite.conf
+SCALELITE_API_PORT=${SCALELITE_API_PORT:-3000}
+
+envsubst '$URL_HOST $SCALELITE_API_PORT' < $nginx_template >/etc/nginx/conf.d/scalelite.conf
 
-envsubst '$URL_HOST' <$nginx_template >/etc/nginx/conf.d/scalelite.conf
```

Comment:
Couple issues here:

1. `$SCALELITE_API_PORT=${SCALELITE_API_PORT:-3000}` would evaluate to `3000=3000` because of shell substitution. I renamed it to `SCALELITE_API_PORT` instead.
1. `envsubst` is ran on `$nginx_template` and then immediately ran again to overwrite the previous changes. In the rendered `conf.d/scalelite.conf` the `$SCALELITE_API_PORT` variable has not be interpolated.